tests/ga/test_collect_logs.py

```diff
@@ -120,6 +122,7 @@ def test_log_collection_is_invoked_with_resource_limits(self):
         self.assertIn("--property=CPUQuota=5%", args[0], "The log collector should have been invoked with a CPU limit")
         self.assertIn("--property=MemoryLimit=30M", args[0], "The log collector should have been invoked with a memory limit")
 
+    @attr('requires_sudo')
```

Comment:
The logcollector handler thread now requires permissions to open the log file, and thus requires sudo.

Reply:
I personally didn't see this as a priority, but I do agree that a mock would be more ideal. If you think this is worth doing, I can add the change.